test/studies/arkouda/sub_test

```diff
@@ -175,6 +185,15 @@ if [ ${CHPL_TEST_ARKOUDA_STOP_AFTER_BUILD} = "false" ]; then
     fi
     test_end
   fi
+
+  # deactivate the venv and cleanup
+  if [ -n "$CHPL_TEST_VENV_DIR" ] && [ "$CHPL_TEST_VENV_DIR" != "none" ]; then
+    deactivate
+  else
+    deactivate
+    rm -rf .venv
```

Comment:
If you take my previous suggestion, don't forget to update the name here, or maybe put it in a local (environment?) variable and just use that.